index.html

```diff
@@ -98,216 +132,10 @@ <h2 class="title-post">Multi-post Stories</h2>
-                        <h2 class="title-post1">Profesional Art Printing Data</h2>
-                        <p class="supporting-text2">
-                            A daily selection of privately personalized reads; no accounts
-                            or sign-ups required. has been the industry's standard
-                        </p>
-                        <nav>
-                            <ul class="tags1">
-                                <li class="html-tag"><a href="#">html</a></li>
-                                <li class="bootstrap-tag"><a href="#">bootstrap</a></li>
-                                <li class="ruby-tag"><a href="#">ruby</a></li>
-                            </ul>
-                        </nav>
-                    </div>
-                    <button class="see-project-btn1">See Project</button>
-                </div>
-                <div class="card-works">
-                    <div class="frame43-right-block">
-                        <h2 class="title-post1">Profesional Art Printing Data</h2>
-                        <p class="supporting-text2">
-                            A daily selection of privately personalized reads; no accounts
-                            or sign-ups required. has been the industry's standard
-                        </p>
-                        <nav>
-                            <ul class="tags1">
-                                <li class="html-tag"><a href="#">html</a></li>
-                                <li class="bootstrap-tag"><a href="#">bootstrap</a></li>
-                                <li class="ruby-tag"><a href="#">ruby</a></li>
-                            </ul>
-                        </nav>
-                    </div>
-                    <button class="see-project-btn1">See Project</button>
-                </div>
-                <div class="card-works" id="last">
-                    <div class="frame43-right-block">
-                        <h2 class="title-post1">Profesional Art Printing Data</h2>
-                        <p class="supporting-text2">
-                            A daily selection of privately personalized reads; no accounts
-                            or sign-ups required. has been the industry's standard
-                        </p>
-                        <nav>
-                            <ul class="tags1">
-                                <li class="html-tag"><a href="#">html</a></li>
-                                <li class="bootstrap-tag"><a href="#">bootstrap</a></li>
-                                <li class="ruby-tag"><a href="#">ruby</a></li>
-                            </ul>
-                        </nav>
-                    </div>
-                    <button class="see-project-btn1">See Project</button>
+                    <button type="submit" class="see-project-btn1" id="multipost-btn">See Project</button>
                 </div>
             </div>
             <div class="grid-container1">
```

Comment:
- [ ] Add a flex class here for example 
` <div class="grid-container1 flex">`

The do a css class of flex
```
.flex {
      display: flex
       }
```
Every component you wish to apply the flex display, just add to the class in html